modules/ztp-image-based-upgrade-procedure-steps.adoc

```diff
@@ -0,0 +1,191 @@
+  oadpContent: <4>
+    - name: oadp-cm
+      namespace: openshift-adp
+  plan: <5>
+  - actions: ["Prep"]
+    rolloutStrategy:
+      maxConcurrency: 2
+      timeout: 2400
+----
+<1> Specify the set of clusters that you want to upgrade.
+<2> Defines the target platform version, the seed image to be used, and the secret required to access the image.
+<3> (Optional) Specify the list of `ConfigMap` resources that contain your custom catalog sources to retain after the upgrade, and your extra manifests to apply to the target cluster that are not part of the seed image.
+<4> Specify the list of `ConfigMap` resources that contain the OADP `Backup` and `Restore` CRs.
+<5> Defines the upgrade plan.
+--
+
+. Apply the `example-group-upgrade.yaml` file by running the following command on the hub cluster:
++
+[source,terminal]
+----
+$ oc apply -f example-group-upgrade.yaml
+----
+
+. Monitor the status updates by running the following command on the hub cluster:
++
+--
+[source,terminal]
+----
+$ oc get ibgu -o yaml
+----
+
+.Example output
+[source,yaml]
+----
+[...]
+status:
+  clusters:
+  - completedActions:
+    - action: Prep
+    name: spoke1
+  - completedActions:
+    - action: Prep
+    name: spoke4
+  - failedActions:
+    - action: Prep
+    name: spoke6
+[...]
+----
+
+This example plan starts with the `Prep` stage only and you append actions to the plan based on the results of the step
+{cgu-operator} adds a label to the clusters to mark if the upgrade succeeded or failed.
+For example, the `lcm.openshift.io/ibgu-prep-failed` is applied to clusters that failed the `Prep` stage.
+
+After investigating the reason for failure, you can append the `AbortOnFailure` step to move the clusters labeled with `lcm.openshift.io/ibgu-<action>-failed` back to the `Idle` stage.
+This deletes any resources on the selected clusters that are related to the upgrade and allows you to retry the upgrade after troubleshooting.
+--
+
+. (Optional) Add the `AbortOnFailure` action to your existing `ImageBasedGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc patch ibgu example-group-upgrade --type=json -p \
+'[{"op": "add", "path": "/spec/plan/-", "value": {"actions": ["AbortOnFailure"], "rolloutStrategy": {"maxConcurrency": 5, "timeout": 10}}}]'
+----
+
+.. Continue monitoring the status updates by running the following command:
++
+[source,terminal]
+----
+$ oc get ibgu -o yaml
+----
+
+. Add the action to your existing `ImageBasedGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc patch ibgu example-group-upgrade --type=json -p \
+'[{"op": "add", "path": "/spec/plan/-", "value": {"actions": ["Upgrade"], "rolloutStrategy": {"maxConcurrency": 2, "timeout": 30}}}]'
+----
+
+. (Optional) Add the `AbortOnFailure` action to your existing `ImageBasedGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc patch ibgu example-group-upgrade --type=json -p \
```

Comment:
Half a suggestion (not sure what description should go inside the angle brackets):
```suggestion
$ oc patch ibgu <some_filename-agnostic_custom_resource_description> --type=json -p \
```